.env.example

```diff
@@ -4,3 +4,4 @@ RUST_LOG=debug
 DB_URL=
 DB_NAME=
 HOST=
+PASSWORD_SALT=
```

Comment:
instead of having a single password_salt for all users can we generate separate salt for each user and store it within the user data. i